deel/influenciae/common/base_influence.py

```diff
@@ -269,30 +269,6 @@ def compute_influence_vector(
 
         return inf_vect_ds
 
-    @abstractmethod
```

Comment:
Why removing it? Not all class should implement it?

Reply:
Because it was already implemented further down. I was just addressing issue #21 ;)